src/router/routes.ts

```diff
@@ -40,10 +42,16 @@ const routes = [
     name: 'Parent',
     component: Parent,
   },
+  {
```

Comment:
It looks like the individual parts of the whole collection had their URI changed,

So they no longer include the contract address in it, but we refer to them by name?

Reply:
Yep, @Kahonnohak asked for that. I have internal mapping between collection name and contract address.